src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegrationInDirectory.tsx

```diff
@@ -200,3 +199,38 @@ const IntegrationFlex = styled('div')`
 const IntegrationItemBox = styled('div')`
   flex: 1;
 `;
+
+const IntegrationStatus = styled((props: {status: ObjectStatus}) => {
+  const {status, ...p} = props;
+  const color = status === 'active' ? theme.success : theme.gray2;
+
+  return (
+    <Tooltip
+      title={t('This Integration has been disconnected from the external provider')}
+      disabled={status === 'active'}
```

Comment:
@NisanthanNanthakumar I would like a tooltip when the integration is active saying the way to disable the integration is by pressing "uninstall".